recording.js

```diff
@@ -212,8 +213,8 @@ exports.startRecording = function(canvases, fileName) {
 
     record.onstop = () => {
       merger.destroy();
-      const blob = new Blob(chunks, { type: 'video/mp4' });
-      save(fileName, blob);
+      const blob = new Blob(chunks, { type: mimType });
+      callback(blob, extension(mimType));

```

Comment:
Could you make `extension(mimType)`  into `mime-types.extension(mimType)`  I couldn't tell if was a build in function or not at first?